sdks/v4-sdk/src/utils/currencyMap.ts

```diff
@@ -0,0 +1,10 @@
+import { Currency } from '@uniswap/sdk-core'
+import { ADDRESS_ZERO } from '../internalConstants'
+
+// Uniswap v4 supports native pools. Those currencies are represented by the zero address.
+// TODO: Figure out if this is how we should be handling weird edge case tokens like CELO/Polygon/etc..
```

Comment:
I believe this has been resolved and that any Currency's isNative value will be set according to how it should be "treated" in the interface